src/mbgl/map/map_context.cpp

```diff
@@ -420,7 +313,6 @@ void MapContext::onResourceLoadingFailed(std::exception_ptr error) {
 }
 
 void MapContext::onSpriteStoreLoaded() {
-    updateAnnotationTilesIfNeeded();
 }
```

Comment:
We don't necessarily need `onSpriteStoreLoaded()` anymore. This was added in https://github.com/mapbox/mapbox-gl-native/commit/0d899df9ddc9a7a9a8bfbac2c4b077ce26da64e9 to address #1675, but this PR obsoletes that.